src/libical/icalcomponent.h

```diff
@@ -146,13 +146,13 @@ LIBICAL_ICAL_EXPORT icalcomponent *icalcompiter_deref(icalcompiter * i);
 /* Working with embedded error properties */
 LIBICAL_ICAL_EXPORT int icalcomponent_check_restrictions(icalcomponent *comp);
 
-/** Count embedded errors. */
+/** Counts embedded errors. */
```

Comment:
"Returns the number of embedded errors"

Reply:
Fixed in later commits copying from the `.c`. It's now 
```
/** @brief Returns the number of errors encountered parsing the data.
 *
 * This function counts the number times the X-LIC-ERROR occurs
 * in the data structure.
 */
```